### PR TITLE
[WIP] Fix config guide topic for performance testing

### PR DESCRIPTION
**Pull request by hguthrie**
The PR corrects the presentation of the Configuration guide topic: [Generate data for performance testing](https://devdocs.magento.com/guides/v2.2/config-guide/cli/config-cli-subcommands-perf-data.html).

- converted HTML table to MD
- cleaned the include file
- the 2.3 version and the 2.2 version were identical, so deleted the duplicate file and created a symlink to 2.2 version
- removed all HTML and replaced with XML code blocks
- removed all glossary links (some were inside of code blocks)

I did not alter the technical content, but I will send a preview to a dev to make sure it presents appropriately.

- [x] technical review
- [x] link check
- [x] editorial review

### CODE REVIEW

File: guides/v2.2/config-guide/cli/config-cli-subcommands-perf-data.md [hunk 1]
```diff
@@ -7,543 +7,419 @@ functional_areas:
...
+customers      | 200    | 2,000  | 2,000              | 5,000        | 10,000
+tax_rates      | 130    | 40,000 | 40,000             | 40,000       | 40,000
+orders         | 80     | 50,000 | 50,000             | 100,000      | 150,000
+
+{:.procedure}
+To run the data generator:
+
+1. Disable all cron jobs before running the data generator. Disabling cron jobs prevents the data generator from performing actions that conflict with active cron jobs and avoids unnecessary errors.
+
+1. Run the data generator using the absolute fie system path and name of a profile.
```
Comment (adifucan) [inline guides/v2.2/config-guide/cli/config-cli-subcommands-perf-data.md]: absolute file system path

File: guides/v2.2/config-guide/cli/config-cli-subcommands-perf-data.md [hunk 2]
```diff
@@ -7,543 +7,419 @@ functional_areas:
   - Setup
 ---
 
-## Overview of performance testing data {#config-cli-perf-overview}
+Using the [Magento Performance Toolkit]({{ site.mage2bloburl }}/{{ page.guide_version }}/setup/performance-toolkit), or another tool, for performance testing requires you to generate a large amount of data, including stores, categories, products, and more. You can adjust the amount of data you create using _profiles_, such as small, medium, large, and extra large.
```
Comment (adifucan) [inline guides/v2.2/config-guide/cli/config-cli-subcommands-perf-data.md]: It's unclear what another tools we are talking about. I guess it's better to eliminate "another tools" from here as we not refer to them.

File: guides/v2.2/config-guide/cli/config-cli-subcommands-perf-data.md (hunk 1 above)
Comment (hguthrie) [reply guides/v2.2/config-guide/cli/config-cli-subcommands-perf-data.md]: Cool! thanks for catching that!

**Comment by hguthrie** [comment]
running tests

**Pull request merged**